src/Examine/LuceneEngine/Search/LuceneSearchQueryBase.cs

```diff
@@ -20,7 +20,8 @@ public abstract class LuceneSearchQueryBase : IQuery, INestedQuery
 
         internal readonly Stack<BooleanQuery> Queries = new Stack<BooleanQuery>();
         public BooleanQuery Query => Queries.Peek();
-        internal readonly List<SortField> SortFields = new List<SortField>();
+
+        public List<SortField> SortFields => new List<SortField>();
```

Comment:
Because this is an expression bodied getter function, it will initialize a new list every time the value is returned. It should use an [auto-property initializer](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-6#auto-property-initializers) and probably also return an `IList<SortField>` instead:

```suggestion
        public IList<SortField> SortFields { get; } = new List<SortField>();
```

Reply:
Yep good catch

Reply:
Have made this change, this error actually causes 3x tests to fail without this update.